src/modules/playground.ts

```diff
@@ -76,17 +75,67 @@ export class PlaygroundModule extends Module {
 		}
 	}
 
+	@listener({ event: 'message' })
+	async onPlaygroundLinkAttachment(msg: Message) {
+		if (msg.author.bot) return;
+		const attachment = msg.attachments.find(a => a.name === 'message.txt');
+		if (msg.author.bot || !attachment) return;
+		const content = await fetch(attachment.url).then(r => r.text());
+		const originalUrl = extractPlaygroundLink(content);
+		if (!originalUrl || originalUrl !== content) return;
```

Comment:
By default, if you write a message in the box and then paste a long playground link, it will only put the paste in `message.txt` and will put the rest of the message in `msg.content`.

Thus IMO shortening an attached file that includes a playground link, but also includes other text, is a rather niche use-case, and would likely cause the rest of the content in `message.txt` to go overlooked.

Reply:
```suggestion
		// By default, if you write a message in the box and then paste a long
		// playground link,wit will only put the paste in message.txt and will
		// put the rest of the message in msg.content
		if (!originalUrl || originalUrl !== content) return;
```

Reply:
Or in the commit message